README.md

```diff
@@ -117,7 +116,7 @@ $ rails new . -d postgresql
 ```
 
 ```
-Your answer...
+"rails new . -d postgresql" creates a new database.  "rails new tunr" creates a new rails application called "tunr" and its file directory.
 ```
```

Comment:
It's important to indicate here that `rails new . -d postgresql` creates a Rails application that (a) uses Postgres as a database and (b) places all Rails files in the current directory.

`rails new tunr` creates a new directory Tunr in the current directory which contains all the Rails files. Because Postgres is not specified as the database of choice, the application defaults to using SQLite as the database management tool.